### PR TITLE
Refactor map styles for forking

### PR DESCRIPTION
**Pull request by alfogrillo**
This PR makes more explicit how to replace a map style (eg. when forking). 
Furthermore it adds UTs for map tiler url builders.

### CODE REVIEW

File: ElementX/Sources/Other/MapLibre/MapTilerStyle.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+/// The style for a map.
+/// Values should be Map Libre style IDs generated with an account where the API key belongs to.
+enum MapTilerStyle: String {
+    case light = "9bc819c8-e627-474a-a348-ec144fe3d810"
+    case dark = "dea61faf-292b-4774-9660-58fcef89a7f3"
+}
```
Comment (Velin92) [inline ElementX/Sources/Other/MapLibre/MapTilerStyle.swift:22]: So I imagine these could be changed according to the user's API settings?

Comment (Velin92) [reply ElementX/Sources/Other/MapLibre/MapTilerStyle.swift:22]: Maybe I would just make the doc a bit more cleare by writing something like:

`// FOR FORKS ONY: replace with your color keys
`
or something like that

**Review by Velin92 — APPROVED** [review]
LGTM

File: ElementX/Sources/Other/MapLibre/MapTilerStyle.swift (hunk 1 above)
Comment (alfogrillo) [reply ElementX/Sources/Other/MapLibre/MapTilerStyle.swift:22]: This are map styles identifiers.

They aren't supposed to be for forks only, we may decide to change them later.

People can:
- Use public styles like `basic-v2` (reported in the doc like an example)
- Using custom styles (like us). If this is the case, the api key and the custom style should belong to the same account.

Does this clarify?

**Pull request merged**